app/views/shared/_terms.html.erb

```diff
@@ -724,20 +774,17 @@ meals.</p>
   <li>Family Automated Screening Tool (FAST): <a href="http://www.michigan.gov/fast">www.michigan.gov/fast</a></li>
   <li>Michigan Department of Health and Human Services (MDHHS): <a href="http://www.michigan.gov/mdhhs">www.michigan.gov/mdhhs</a></li>
   <li>Applying for Assistance: <a href="http://www.michigan.gov/dhs-applicationprocess">www.michigan.gov/dhs-applicationprocess</a></li>
-  <li><a href="http://www.michigan.gov/dhs-applicationprocess"></a></li>
   <li>Cash Assistance: <a href="http://www.michigan.gov/dhs-cash">www.michigan.gov/dhs-cash</a></li>
-  <li ><a href="http://www.michigan.gov/dhs-cash"></a></li>
-    <li >Emergency Services: <a href="http://www.michigan.gov/dhs-ser">www.michigan.gov/dhs-ser</a></li><li
-><a href="http://www.michigan.gov/dhs-ser"></a></li><li
-    >Food Assistance: <a href="http://www.michigan.gov/foodstamps">www.michigan.gov/foodstamps"</a></li>
-    <li >Medical Services: <a href="http://www.michigan.gov/dhs-medical">www.michigan.gov/dhs-medical</a></li>
+  <li>Emergency Services: <a href="http://www.michigan.gov/dhs-ser">www.michigan.gov/dhs-ser</a></li>
+    <li>Food Assistance: <a href="http://www.michigan.gov/foodstamps">www.michigan.gov/foodstamps"</a></li>
```

Comment:
Appears to have indented accidentally starting on this line. This line also has a dangling `"` before `</a>`.

Reply:
good catch!

fixed at a3e34ad0